test/rules/max-line-length/ignore-imports/test.ts.lint

```diff
@@ -0,0 +1,29 @@
+import { obj1, obj2, obj3, obj4 } from 'my-favorite-module/with/lots/of/deep/nested/modules';
+import {
+    obj1,
+    obj2,
+    obj3,
+    obj4,
+} from 'my-favorite-module/with/lots/of/deep/nested/modules';
+
+import {
+    obj1, obj2, obj3, obj4, just, trying, to, be, a, rebel, here
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ [Exceeds maximum line length of 50]
```

Comment:
This shouldn't be ignored. If they choose to do

```
import { obj1, obj2, obj3, obj4, just, trying, to, be, a, rebel, here } from '...';
```

then that is ok, but the moment they decide to break the line then we have options to prevent exceeding the line limit. I usually prefer to place each import on its own line when using this form:


```
import {
  obj1,
  obj2,
  obj3,
  obj4,
  just,
  trying,
  to,
  be,
  a,
  rebel,
  here,
} from '...';
```

I decided to go with this approach because in the past I have seen a really long list increasing on the same line and it gets hard to see the changes. So really the rule should just apply to the string part of the import which we don't have much control over.